students/medrek_tomasz/lesson_06_dicts_tuples_sets_args_kwargs/access_rights.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+
+OPERATIONS_PERMS = {'write': 'W', 'read': 'R', 'execute': 'X'}
+access_rights_database = {}
+
+
+def check_access_right(operation, file):
+    if file in access_rights_database.keys()\
```

Comment:
You can use: if access_rights_database.get(file)
http://www.tutorialspoint.com/python/dictionary_get.htm